src/bug-filing/common/create-issue-details-builder.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { title } from '../../content/strings/application';
+import { IssueDetailsBuilder } from './issue-details-builder';
+import { MarkupFactory } from './markup/markup-factory';
+
+export const createIssueDetailsBuilder = (markup: MarkupFactory): IssueDetailsBuilder => {
+    const getter = (environmentInfo: EnvironmentInfo, data: CreateIssueDetailsTextData): string => {
+        const result = data.ruleResult;
+
+        const text = [
+            `${markup.bold('Issue')}: ${markup.snippet(result.help)} (${markup.link(result.helpUrl, result.ruleId)})`,
```

Comment:
>bold [](start = 22, length = 4)

Is "bold" actually the ideal thing for us to be using here? I think we had gotten guidance from design when we made our PR/Bug templates that `####` was the preferred section heading distinguisher in markdown; would it be better to do a `markup.sectionHeader` here and have the 2 factories use `h4` and `####` instead?

Reply:
This is an interesting point. I'm not sure what the context and scope of that guidance is, and I'm not sure it affects content we generate but we don't directly render.
That being said, I'll explore using your suggestions, see how it behave/looks and decide base on what I gather.

Reply:
@ferBonnin: Dan has a good point here about making the content accessible by using headers instead of just bold text. I made the change and update the screenshots. Can you have a look? Let me know what you think. Thanks